config/quota-management-list-configuration.yaml

```diff
@@ -5,25 +5,64 @@
 # In the future it may support service accounts outside RH.
 # The structure of registered service accounts is:
 #       - username: is the account of the user. The username must be unique
-#       - max_allowed_instances: is the maximum number of instances this user can create.
+#       - max_allowed_instances: is the maximum number of instances of a given instance_type/billing_model pair this user can create.
+#         This value is used if `allowed` field (inside `billing_model`) is 0 or omitted.
 #         Defaults to the global value of `max-allowed-instances` which has different values for distinct environments.
+#       - granted_quota: the configuration of the quota granted to the user. The configuration is defined as follows:
+#         - instance_type_id: the id of the instance type for which we are configuring the quota
+#         - billing_models: the list of billing models that we are going to configure.
+#           The set of accepted billing models, depends on the KafkaBillingModel configuration.
```

Comment:
```suggestion
#         - billing_models: the list of billing models that we are going to configure.
#           The set of accepted billing models, depends on the KafkaBillingModels defined per instance type in the 'supported_instance_types' configuration. 
```

Reply:
Can we also make a note here on the defaults given to the `billing_models` when its set to an empty array